rnndb/bus/pci.xml

```diff
@@ -406,6 +406,23 @@
 	<reg32 offset="0x1c8" name="P2P_READ_REPLY_ADDR_LOW" stride="8" length="3" variants="G80:GF100"/>
 	<reg32 offset="0x1cc" name="P2P_READ_REPLY_ADDR_HIGH" stride="8" length="3" variants="G80:GF100"/>
 	<!-- XXX: 420+ is AER -->
+	<reg32 offset="0x460" name="CONFIG_LINK" variants="G84:GK104">
+		<bitfield pos="0" name="COMMIT"/>
+		<bitfield pos="1" name="UNK1"/>
+		<bitfield pos="2" name="UNK2" variants="G92-"/>
+		<bitfield pos="3" name="UNK3" variants="GT218-"/>
+		<bitfield low="4" high="5" name="SPEED">
```

Comment:
On my chip, it's b06c2220.

I bet it's just 3 fields with speeds of various things in lspci, 4:7, 8:11, 12:15.